app/views/posts/index.html.erb

```diff
@@ -1,57 +1,67 @@
 <div class="d-flex flex-column align-items-center gap-4 my-5">
     <div class="container d-flex w-75 p-0">
-        <a href="<%= user_path(@user) %>"><img class="rounded-circle bg-white p-2 border me-4" src="<%= @user.photo %>" alt="User Photo"></a>
+        <a href="<%= user_path(@user) %>">
+            <img class="rounded-circle bg-white p-2 border me-4" src="<%= @user.photo %>" alt="User Photo">
+        </a>
         <div class="border rounded-5 w-100 bg-white p-4 d-flex flex-column justify-content-between">
             <div class="w-25 m-0">
-                <h2 class="text-decoration-none fs-2 text-decoration-underline-hover text-primary"><%= link_to @user.name, user_path(@user), class: "text-decoration-none" %></h2>
+                <h2 class="text-decoration-none fs-2 text-decoration-underline-hover text-primary">
+                    <%= link_to @user.name, user_path(@user), class: "text-decoration-none" %>
+                </h2>
                 <p class="user-posts text-end me-5 fs-5">Number of posts: <%= @user.posts_counter %></p>
             </div>
-            <%= link_to "Back to Home", root_path, title:"To Home Page", class: "text-decoration-none fs-4 text-decoration-underline-hover text-primary align-self-end" %>
+            <div class="align-self-end d-flex align-items-center gap-3">
+                <% if @current_user == @user %>
+                    <%= link_to "Create New Post", new_user_post_path(@user), class: "btn btn-primary fs-3 px-4 py-2 rounded-pill" %>
```

Comment:
🚀  Fantastic job on the Blog app 💯  
Thank you for putting an effort into submitting a well designed, well-documented, clean , and well structured project.  You went above and beyond and delivered an an excellent application.

![image](https://github.com/grauJavier/blog-app-ruby/assets/38088818/08b23d05-d3a2-4e5f-a1e3-1bb0a01fcd12)

### Summary:
- The UI of the application is visually appealing and user-friendly. You have clearly paid attention to the design and layout, making it easy for users to navigate and interact with the app. 
- The forms are well-designed and intuitive, allowing users to create new posts, comments, and likes effortlessly.
- The codebase is clean and well-organized, making it easy to understand and maintain. It's evident that you have a good understanding of Rails conventions and have applied them effectively.
- Great implementation of Restful routes for creating new posts, comments, and likes, following the RESTful principles. 
- The controller actions are well-written and perform the necessary operations efficiently. It's evident that you have a good grasp of controller logic and has implemented it effectively.
- Kudos! on adding adequate data on the `seed.rb` file making it easy to populate and test the database.

Overall, I am highly impressed with you work on the Blog app application. Thank you for demonstrating a strong understanding of Rails concepts and have implemented the required functionality with great attention to detail. 

Congratulations you on the outstanding effort and the successful completion of this milestone. 🔥 
Keep up the excellent work! 🥂